project.clj

```diff
@@ -35,18 +34,30 @@
   ;; depend on this source jar using a :classifier in their :dependencies.
   :classifiers [["sources" :sources-jar]]
 
-  :profiles {:dev {:dependencies [[org.clojure/clojure]]
+  :profiles {:dev {:dependencies [[org.clojure/clojure]
+                                  [org.bouncycastle/bcpkix-jdk15on]]
+                   :resource-paths ["test-resources"]}
+
+             ;; per https://github.com/technomancy/leiningen/issues/1907
+             ;; the provided profile is necessary for lein jar / lein install
+             :provided {:dependencies [[org.bouncycastle/bcpkix-jdk15on]]
+                        :resource-paths ["test-resources"]}
+
+             :fips {:dependencies [[org.clojure/clojure]
+                                   [org.bouncycastle/bcpkix-fips]
+                                   [org.bouncycastle/bc-fips]]
                    :resource-paths ["test-resources"]}
 
              :sources-jar {:java-source-paths ^:replace []
                            :jar-exclusions ^:replace []
                            :source-paths ^:replace ["src/clojure" "src/java"]}}
 
-  :plugins [[lein-parent "0.3.1"]
+  :plugins [[lein-parent "0.3.5"]
```

Comment:
I made a similar note in the clj-http-client PR, it'd be nice, but far from necessary to update this to 0.3.7 on Monday.

Reply:
We can do that!